	- pass CFLAGS manually since using $(M) from top-level Makefile
	  causes passing CFLAGS twice in implicit rules
--- maradns-2.0.09.orig/tcp/Makefile.authonly	2006-07-18 05:50:44.000000000 +0200
+++ maradns-2.0.09/tcp/Makefile.authonly	2014-06-19 03:21:59.379141091 +0200
@@ -25,11 +25,11 @@ clean:
 	rm -f core $(EXECS) $(TCPOBJS) *.exe
 
 getzone: getzone.c $(OBJECTS) 
-	$(CC) $(FLAGS) -o getzone getzone.c $(OBJECTS)
+	$(CC) $(CFLAGS) $(FLAGS) -o getzone getzone.c $(OBJECTS)
 
 fetchzone: fetchzone.c $(OBJECTS) 
-	$(CC) $(FLAGS) -o fetchzone fetchzone.c $(OBJECTS) 
+	$(CC) $(CFLAGS) $(FLAGS) -o fetchzone fetchzone.c $(OBJECTS) 
 
 zoneserver: zoneserver.c zoneserver_en.h $(OBJECTS)
-	$(CC) $(FLAGS) -DVERSION=\"$(VERSION)\" -o zoneserver zoneserver.c $(OBJECTS) $(RNGOBJS)
+	$(CC) $(CFLAGS) $(FLAGS) -DVERSION=\"$(VERSION)\" -o zoneserver zoneserver.c $(OBJECTS) $(RNGOBJS)
 
